test/fixtures/tls-connect.js

```diff
@@ -6,6 +6,9 @@
 const common = require('../common');
 const fs = require('fs');
 const join = require('path').join;
+// Check if Node was compiled --without-ssl and if so exit early
+// as the require of tls will otherwise throw an Error.
+checkCrypto();
```

Comment:
Can you fix up the tests that include this file?  They call this function too but that's unnecessary with this change.  You can probably stop exporting checkCrypto() altogether and just inline it here.